src/components/NavBar.tsx

```diff
@@ -0,0 +1,18 @@
+import { memo } from 'react';
+import { PageNavLink } from './PageNavLink';
+
+export const NavBar = memo(() => (
```

Comment:
> since the HomePage component appears to be a simple static component without any dynamic behavior, memoizing it may not provide any significant benefits